metomi/rosie/ws_client_cli.py

```diff
@@ -295,13 +350,7 @@ def _display_maps(opts, ws_client, dict_rows, url=None):
     """Display returned suite details."""
     report = ws_client.event_handler
 
-    try:
-        terminal_cols = int(ws_client.popen("stty", "size")[0].split()[1])
-    except (IndexError, RosePopenError, ValueError):
-        terminal_cols = None
-
-    if terminal_cols == 0:
-        terminal_cols = None
```

Comment:
The `stty size` command just does not work when run via the Rose Popener module.

I did try hacking it, but I couldn't figure out what Popener was doing to aggravate stty and gave in.